internal/pkg/runtime/engine/apptainer/container_linux.go

```diff
@@ -1739,6 +1746,17 @@ func (c *container) addUserbindsMount(system *mount.System) error {
 	defaultFlags := uintptr(syscall.MS_BIND | c.suidFlag | syscall.MS_NODEV | syscall.MS_REC)
 
 	for _, b := range c.engine.EngineConfig.GetBindPath() {
+		if strings.HasPrefix(b.Destination, "/.singularity.d/libs") {
```

Comment:
please add a comment explaining context and why this is required

Reply:
comment added